game/splash.rpy

```diff
@@ -375,7 +375,7 @@ label splashscreen:
         # software so the game can enable Let's Play Mode automatically and notify
         # the user about it if extra settings are enabled.
         if extra_settings:
-            if process_check(["obs32.exe", "obs64.exe", "obs.exe", "xsplit.core.exe", "livehime.exe", "pandatool.exe", "yymixer.exe", "douyutool.exe", "huomaotool.exe", "BlueTool"]):
```

Comment:
What was this "BlueTool"

Reply:
Leftover Test Code for running the process list on MacOS.